src/test/java/tests/MultiModelTest.java

```diff
@@ -23,7 +23,7 @@ public static void setup() {
         UI.status = new StatusUIStub();
     }
 
-    MultiModel multiModel = new MultiModel(new Preferences(true));
+    MultiModel multiModel = new MultiModel(mock(Preferences.class));
```

Comment:
Sure. Can add it into documentation? It's quite basic in terms of unit testing but code copying might result in people using test pref when they should use mocking and vice versa.